src/mon/PGMap.cc

```diff
@@ -3273,15 +3302,20 @@ int process_pg_map_command(
 	if (what.count("pgs")) {
 	  pg_map.dump_pg_stats(ds, false);
 	  header = false;
+          omap_stats_note_required = true;
 	}
 	if (what.count("pools")) {
 	  pg_map.dump_pool_stats(ds, header);
+          omap_stats_note_required = true;
 	}
 	if (what.count("osds")) {
 	  pg_map.dump_osd_stats(ds);
 	}
       }
       odata->append(ds);
+      if (omap_stats_note_required) {
+        odata->append(omap_stats_note);
```

Comment:
Not a big deal, but maybe this should append to ds prior to the odata append?

Reply:
Will do.

Reply:
Actually, doing it this way just eats my new line between the output and my note so I might stick with the original @gregsfortytwo